monosi/anomalies/__init__.py

```diff
@@ -13,11 +13,20 @@ class AnomalyDetector:
 
 class ZScoreAnomalyDetector(AnomalyDetector):
     @classmethod
-    def _mean(cls, values: List[Any]):
+    def _remove_nulls(cls, stats: List[MetricStat]):
+        filtered_stats = []
+        for stat in stats:
+            if stat.value is not None:
+                filtered_stats.append(stat)
+
+        return filtered_stats
```

Comment:
Not related to the class, consider making a separate function or directly in method using
```
list(filter(bool, values))
```

or similar short pythonic expression

Reply:
Using this expression to filter the actual stats list instead of just the values 
```[stat for stat in stats if stat.value != None]```